docs/user_guides/debugging/features.rst

```diff
@@ -1,7 +1,7 @@
 Supported GDB Features
 ======================
 
-Currently, the following debugging features are available:
+Currently the following debugging features are available:
```

Comment:
I think the `comma` was Ok in this case. But, I prefer we drop the adverb completely and change the sentence to: "The following debugging..."

Reply:
Fixed